# TransformationMatrix.msg
string source_frame
string registration_frame
float64[16] matrix
time stamp